graphql_api/app/graphql/queries/insureds_query.rb

```diff
@@ -0,0 +1,11 @@
+module Queries
+  class InsuredsQuery < Queries::BaseQuery
+    type [Types::InsuredType], null: false
+
+    def resolve
+      response = HTTParty.get("http://rest_api:5000/insureds",
```

Comment:
O que você acha de usar uma variável de ambiente para representar a url da rest-api, pois caso mude algum dia, você terá que alterar em diversos lugares.
```suggestion
      response = HTTParty.get("#{ENV['REST_API_URL']}/insureds",
```

Reply:
Boa, de fato, dá pra fazer mais melhorias nesse sentido, acredito que nos próximos passos eu deva implementar essas mudanças e abrir um PR para essas melhorias. Acha que devo priorizar o update do pagamento, para fazer jus as rotas e actions que já deixei prontas referente ao update, destroy e edit ou acha que devo priorizar essas refatorações?